airbyte-integrations/connectors/source-qualaroo/bin/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.7.11-alpine3.14 as base
+
+# build and load all requirements
+FROM base as builder
+WORKDIR /airbyte/integration_code
+
+# upgrade pip to the latest version
+RUN apk --no-cache upgrade \
+    && pip install --upgrade pip
+
+COPY setup.py ./
+# install necessary packages to a temporary folder
```

Comment:
are these files duplicated ? Looks like bin folder is not needed

Reply:
yeah used another connector as boilerplate, forgot to remove